Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -0,0 +1,31 @@
+ BTWK_story_event.0001.a:0 "Send word to my son, Arthur, though it pains me so..."
+
+ BTWK_story_event.0002.t:0 "Uther's Sickness"
+ BTWK_story_event.0002.desc:0 "Never being quite able to shake your ailments, your health is nearing its breaking point. It's a surprise you've managed to hold on this long, although you'd rather you hadn't. The pneumonia has left you completely incapacitated and bedridden. Unable to even perform the most basic of tasks, you have entrusted (getdiarch) to serve as regent in your dying days."
+ BTWK_story_event.0002.a:0 "I have managed all I can. May the Gods be merciful."
+
+ BTWK_story_event.0005.t:0 "Death of King Uther Pendragon"
+ BTWK_story_event.0005.desc:0 "Uther Pendragon, the King of Dumnonia, is dead. The only legitimate heir to the throne is his clubfooted grandson Mordred, a mere infant. Upon receiving news of his father's death, Uther's bastard son Arthur has returned from the Kingdom of Benoic and has been appointed Lord of Durnac by the current regent."
+ BTWK_story_event.0005.a:0 "Hail to King Mordred and Lord Arthur!"
+ BTWK_story_event.0005.b:0 "Let us hope Arthur drives the Saxons back to whence they came!"
+ BTWK_story_event.0005.c:0 "They are weak. We will conquer their lands, no matter their leader."
+ BTWK_story_event.0005.d:0 "This is interesting news."
+
+ BTWK_story_event.0006.t:0 "From One Pendragon to Another"
+ BTWK_story_event.0006.desc:0 "Your grandfather, King Uther Pendragon, has died. Prince Arthur, his bastard son and your uncle, has returned to Dumnonia from the Kingdom of Benoic upon hearing of Uther's death, where he had been sent to assist King Ban in his wars against the Franks. Upon his return, your current regent appointed him the Prince of Durnac, per succession arrangements previously laid out by Uther. He has been hailed, like his father, as #italic Pendragon #. "
```

Comment:
"per succession arrangements previously laid out by Uther" seems a bit overly formal - maybe "as previously arranged", or even leave out that phrase altogether